cypress/integration/app_admin/app_admin.js

```diff
@@ -0,0 +1,45 @@
+import { Given, When, Then, Before, After } from 'cypress-cucumber-preprocessor/steps';
+
+const USER_NAME = 'test@unit.no';
+const APP_ADMIN_ROLE = 'App-admin';
+
+Before(() => {
+  cy.deleteUser(USER_NAME).then(() => {
+    cy.addUser(USER_NAME, APP_ADMIN_ROLE).then((userId) => {

```

Comment:
Spørsmål: er det mulig å bruke async/await for å slippe nesting av opptil flere `.then()`-kall?
Type
```
await cy.deleteUser(USER_NAME);
const userId = await cy.addUser(USER_NAME, APP_ADMIN_ROLE);
(...)
```

Reply:
Nei, måten Cypress funker på gjør at det blir feil rekkefølge på ting. Prøvde med async/await først og det gikk rett og slett ikke.